host_core/lib/host_core/host.ex

```diff
@@ -87,6 +88,32 @@ defmodule HostCore.Host do
     {:reply, labels, state}
   end
 
+  @impl true
+  def handle_info({:do_stop, _timeout_ms}, state) do
```

Comment:
Super nit: I’ve been defining the timeout in seconds

Reply:
Everything everywhere else in wasmCloud uses milliseconds. Wherever not explicitly mentioned, the assumption is ms.